tools/eslint-rules/project.json

```
@@ -10,5 +10,6 @@
         "jestConfig": "tools/eslint-rules/jest.config.ts"
       }
     }
-  }
+  },
+  "tags": ["tools"]
```

as we are adding it lets follow our pattern:

```suggestion
  "tags": ["platform:node", "tools"]
```

we need to update all/some monorepo projects `project.json` files with this tags if  applicable:

**required:**
- packages/eslint-plugin/project.json

**maybe?:**
- packages/webpack-utilities/project.json
- packages/react-conformance/project.json
- packages/react-components/react-conformance-griffel/project.json
- others...